packages/system/src/column.ts

```diff
@@ -37,8 +36,8 @@ const columnMappings: Record<ColumnKeys, string> = {
   columns: "columns",
 };
 
-const converters: Partial<Record<ColumnKeys, ValueConverter>> = {
-  columnGap: spaceConverter,
+const converters: Partial<Record<ColumnKeys, typeof toCssUnit>> = {
```

Comment:
Is it possible to move the `ValueConverter` type to `system/src/types.ts` instead of removing it? I thought we should use that type here.